src/Theme/AppThemeProvider.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { createMuiTheme, ThemeProvider } from "@material-ui/core/styles";
+
+const theme = createMuiTheme({
```

Comment:
move this object to a theme file so that this provider just takes care of providing the theme and not containing the real thing.
This can be useful if the theme file grows